<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGCMissionDoRepeatServo</class>
 <widget class="QWidget" name="QGCMissionDoRepeatServo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1547</width>
    <height>27</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,10,10,10,0">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QDoubleSpinBox" name="param1SpinBox">
     <property name="minimumSize">
      <size>
       <width>135</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>135</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Mission parameter #1</string>
     </property>
     <property name="statusTip">
      <string>Mission parameter #1</string>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="prefix">
      <string> Servo </string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>-2147483647.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2147483647.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="param2SpinBox">
     <property name="minimumSize">
      <size>
       <width>135</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>135</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Mission parameter #2</string>
     </property>
     <property name="statusTip">
      <string>Mission parameter #2</string>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="prefix">
      <string>PWM </string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>900.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2100.000000000000000</double>
     </property>
     <property name="value">
      <double>1500.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="param3SpinBox">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Mission parameter #3</string>
     </property>
     <property name="statusTip">
      <string>Mission parameter #3</string>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="prefix">
      <string>Repeat </string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2147483647.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="param4SpinBox">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Mission parameter #4</string>
     </property>
     <property name="statusTip">
      <string>Mission parameter #4</string>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="prefix">
      <string>Delay (s) </string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>-2147483647.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2147483647.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
